app/controllers/katello/api/v2/content_view_versions_controller.rb

```diff
@@ -99,6 +99,11 @@ def export_histories
                         :template => '../../../api/v2/content_view_version_export_histories/index')
     end
 
+    api :GET, "/content_view_versions/export_api_status", N_("true if the export api is pulp3 ready and usable")
```

Comment:
I'd personally add text in the description to the effect of "This API is intended for use by hammer-cli only" *just in case*

Reply:
updated